.github/workflows/CI-python.yml

```diff
@@ -25,9 +24,6 @@ jobs:
           [ubuntu-20.04, ubuntu-latest, macos-latest, windows-latest]
         pythonVersion: ["3.6", "3.7", "3.8", "3.9"]
         exclude:
-          - packageDirectory: "rai_core_flask"
-            operatingSystem: macos-latest
-            pythonVersion: "3.9"
           - operatingSystem: ubuntu-20.04
```

Comment:
Can we remove the setting of fail_fast flag as well? I don't think we should run tests that will fail anyways.

Reply:
I suppose for this particular one we can turn it off. I would still like to keep it for the notebook e2e tests, though. Those take ages (1h+) and one of them fails semi-frequently. I'll remove it for the python one.

Reply:
great. Thanks.